### PR TITLE
[Merged by Bors] - doc: fix the docs on `ModelTheory/*`

### PR DESCRIPTION
**Pull request by Komyyy**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/ModelTheory/Basic.lean [hunk 1]
```diff
@@ -29,16 +29,10 @@ structures.
 * A `FirstOrder.Language.Embedding`, denoted `M ↪[L] N`, is an embedding from the `L`-structure `M`
   to the `L`-structure `N` that commutes with the interpretations of functions, and which preserves
   the interpretations of relations in both directions.
-* A `FirstOrder.Language.ElementaryEmbedding`, denoted `M ↪ₑ[L] N`, is an embedding from the
```
Comment (jcommelin) [inline Mathlib/ModelTheory/Basic.lean:32]: Where did this bullet point move? Or why is it dropped?

Comment (Komyyy) [reply Mathlib/ModelTheory/Basic.lean:32]: `ElementaryEmbedding` is defined in `ModelTheory.ElementaryMaps`, not in `ModelTheory.Basic`.

Comment (Komyyy) [reply Mathlib/ModelTheory/Basic.lean:32]: The corresponding explanation is already in `ModelTheory.ElementaryMaps`.

Comment (jcommelin) [reply Mathlib/ModelTheory/Basic.lean:32]: Thanks :+1:

**Comment by jcommelin** [comment]
Thanks :tada:

bors merge

**Comment by fpvandoorn** [comment]
bors crashed, so I'm re-borsing

bors merge

**Pull request closed without merging**